airbyte-integrations/connectors/source-google-ad-manager/sample_files/configured_catalog.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
move this file to `integration_tests/` folder.

Reply:
okay.